test-result-summary-client/src/utils/Utils.js

```diff
@@ -2,6 +2,22 @@ const order = (a, b) => {
     const collator = new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' });
     return collator.compare(a, b);
 }
+const getInfoFromBuildName = (buildName) => {
+    const regex = /^Test_openjdk(\w+)_(\w+)_(\w+).(.+?)_(.+?_.+?(_xl)?)(_.+)?$/i;
+    const tokens = buildName.match(regex);
+    let _, jdkVersion, jdkImpl, level, group, platform;
+    if (Array.isArray(tokens) && tokens.length > 5) {
+        return {
+            _: tokens[0], 
+            jdkVersion: tokens[1], 
+            jdkImpl: tokens[2], 
+            level: tokens[3], 
+            group: tokens[4], 
+            platform: tokens[5]
+        }
+    }
+    return {_, jdkVersion, jdkImpl, level, group, platform};

```

Comment:
Why we are returning undefined values? We can just return `null ` if there is no mtach.

```
if (Array.isArray(tokens) && tokens.length > 5) {
  const [_, jdkVersion, jdkImpl, level, group, platform] = tokens;
  return {jdkVersion, jdkImpl, level, group, platform};
}
return null;
```

Reply:
Values came from`const [_, jdkVersion, jdkImpl, level, group, platform] = tokens;`  are read only, thus could not be exported.

Reply:
I am not sure I understand the above error. Please try again